api/config/config_local.json

```diff
@@ -29,5 +29,8 @@
     "adminIdVerification": "stable-environment-manual-id-verification-requests@fake-research-aou.org",
     "supportGroup": "all-of-us-workbench-eng@googlegroups.com",
     "verifiedSendingAddress": "all-of-us-workbench-eng@googlegroups.com"
+  },
+  "mandrill": {
+    "apiKey": "test-api-key-should-go-here"
```

Comment:
The key should not be here. Look at `CloudStorageServiceImpl` for how we're grabbing the current mailchimp key. We should do the same thing for the mandrill key.

Reply:
kk